db/seeds/app.js

```diff
@@ -17,4 +18,19 @@ app.get("/api/topics", getTopics);
 
 app.get("/api", getApi);
 
+app.get("/api/articles", getArticlesById);
```

Comment:
This is the wrong endpoint, please look into parameters. This should look like this app.get("/api/articles/:article_id", getArticlesById); Let me know if it is not clear, but please look into the notes or the sprints from last week first 🙂